metabase/metabase
# fix: padding not working on <Modal> component

### Description

The default prop for the padding, set on the ModalBody, is overriding the `padding` prop passed to the `<Modal>` component. Basically the padding passed to `<Modal>` is ignored

### How to verify

Make a modal with very high padding such as
```tsx
<Modal
  padding="100px"
>
```
see that it's ignored
### Demo

This is with an very an exaggerated padding, but it's the same even for spacing values from the theme
Before:
![image](https://github.com/metabase/metabase/assets/1914270/6161a4eb-d710-4aa8-9ae1-0b772c4c5ebf)

(Notice the two padding values in the same class in the inspector)

After:
<img width="992" alt="image" src="https://github.com/metabase/metabase/assets/1914270/55fe6b93-3583-402b-a941-6e2b626cb6a8">

Now there's only the correct one
